pkg/credentials/aws/aws.go

```diff
@@ -209,6 +209,69 @@ func DefaultStsProvider(sess *session.Session) stsiface.STSAPI {
 // to missing or expired credentials
 type PopulateFn func(*AuthSAML) error
 
+//type AuthMachine struct {
```

Comment:
Perhaps put in another branch instead of letting these comments go to master?

Reply:
Yeah, I've removed them from this branch